contributed_definitions/NXellipsometry.nxdl.xml

```diff
@@ -0,0 +1,839 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="application" extends="NXobject" name="NXellipsometry" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <symbols>
+        <doc>
+             Variables used throughout the document, e.g. dimensions and important
+             parameters
+        </doc>
+        <symbol name="N_wavelength">
+            <doc>
+                 Size of the energy or wavelength vector used, the length of
+                 NXinstrument/spectrometer/wavelength array
+            </doc>
+        </symbol>
+        <symbol name="N_variables">
+            <doc>
+                 How many variables are saved in a measurement. e.g. 2 for Psi and
+                 Delta, 16 for Mueller matrix elements, 15 for normalized Mueller
+                 matrix, 3 for NCS, the length of NXsample/column_names
+            </doc>
+        </symbol>
+        <symbol name="N_angles">
+            <doc>
+                 Number of incident angles used, the length of
+                 NXinstrument/angle_of_incidence array
+            </doc>
+        </symbol>
+        <symbol name="N_p1">
+            <doc>
+                 Number of sample parameters scanned, if you varied any of the
+                 parameters such as temperature, pressure, or pH, the maximal length of
+                 the arrays specified by NXsample/environment_conditions/SENSOR/value
+                 if it exists.
+            </doc>
+        </symbol>
+        <symbol name="N_time">
+            <doc>
+                 Number of time points measured, the length of NXsample/time_points
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+         Ellipsometry, complex systems, up to variable angle spectroscopy.
+         
+         Information on ellipsometry is provided, e.g. in:
+         - H. Fujiwara, Spectroscopic ellipsometry: principles and applications,
```

Comment:
This is a markdown-style bullet list.  Change to reST:

- add blank lines before and after the bullet list
- change `-` to `*`